py/torch_tensorrt/dynamo/lowering/passes/__init__.py

```diff
@@ -0,0 +1,27 @@
+from typing import Callable
+
+import torch
+from torch.fx.passes.pass_manager import PassManager
+
+from .constant_folding import constant_fold
+from .repair_input_as_output import repair_input_as_output
+
+ATEN_LOWERING_PASSES = PassManager.build_from_passlist(
+    [
+        constant_fold,
+        repair_input_as_output,
+    ]
+)
```

Comment:
`dump_current_lowering_passes` + `add_lowering_pass` takes an optional index arg to specify where in the list the lowering pass gets placed.